pylint/checkers/base/basic_checker.py

```diff
@@ -168,7 +168,8 @@ class BasicChecker(_BasicChecker):
             "exec-used",
             'Used when you use the "exec" statement (function for Python '
             "3), to discourage its usage. That doesn't "
-            "mean you cannot use it !",
+            "mean you cannot use it! "
+            "It's dangerous to use this function for a user input.",
```

Comment:
```suggestion
```

Reply:
Yes, looks better. The `basic_checker.py` has been updated with your text.